internal/cmd/open.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+
+	"cdr.dev/coder-cli/coder-sdk"
+)
+
+func openCmd() *cobra.Command {
+	var (
+		workspaceName string
+		user          string
+	)
+
+	cmd := &cobra.Command{
+		Use:     "open",
+		Short:   "Open Coder workspace IDE",
+		Long:    "Open Coder workspace IDE in your default browser",
+		Example: "coder open --workspace dev",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			client, err := newClient(ctx, true)
+			if err != nil {
+				return err
+			}
+
+			if workspaceName == "" {
+				return xerrors.Errorf("Missing required parameter: --workspace")
+			}
+
+			workspaces, err := getWorkspaces(ctx, client, user)
```

Comment:
If the workspace isn't running, should we start it? Or should we check the status and emit an error message if it's off?

Reply:
I like the idea of just starting it for them. 
I assume if they're running "coder workspace open" they either assume its already built or ultimately _do_ want it built, so we might as well just automate that step